lib/rbeapi/api/system.rb

```diff
@@ -63,6 +63,7 @@ def get
         response = {}
         response.merge!(parse_hostname(config))
         response.merge!(parse_iprouting(config))
+        response.merge!(parse_timezone(config))
```

Comment:
Can you update the method document to show the timezone being returned (above line 56)?